### DISCUSSION TITLE
when i add cooperate email address for my github account it shows already in use

### DISCUSSION THREAD

**Discussion by piseprashant** [community/community#118576] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

when i add cooperate email address for my github account it shows already in use. How to detect where my cooperate email address git address is in use?

**Comment by jfullstackdev** [comment]
I don't know if this trick will work, just trying to help

Try to reset the password for the account using your corporate email address. Go to the GitHub login page and click on "Forgot password". Enter your corporate email address and click on "Send password reset email". You should receive an email with instructions to reset your password. This will also include the username of the account associated with the email.

**Comment by ebndev** [comment]
Hi [piseprashant](https://github.com/piseprashant)! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)
